std/process.d

```diff
@@ -1451,7 +1451,7 @@ void kill(Pid pid, int codeOrSignal)
     }
     else version (Posix)
     {
-        import core.sys.posix.signal;
+        import core.sys.posix.signal : kill;
         if (kill(pid.osHandle, codeOrSignal) == -1)
```

Comment:
This code relied on `import core.sys.posix.signal;` hiding the upscope `kill()`.